### ISSUE TITLE
TextView with markupEnabled in CollectionView cell blocks "select" event under Android

### ISSUE THREAD

**Issue by ifurnadjiev** [eclipsesource/tabris-js#1737]
### Problem description

If you have CollectionView cell with TextView in it, where markupEnabled is set to true, tap on this TextView will not trigger `select` event on the CollectionView. Without markupEnabled everything works. It seems that TextView with markupEnabled  consumes tap events.

Works as expected under iOS.

### Expected behavior

Tap on TextView with markupEnabled set to true inside CollectionView cell triggers `select` event.

### Environment

- Tabris.js version: 2.6.1
- Device: Samsung Galaxy S9
- OS: Android 8.0.0

### Code snippet
Reproducible with CollectionView snippet if you add markupEnabled to the TextView
https://github.com/eclipsesource/tabris-js/blob/v2.6.1/snippets/collectionview.js

**Comment by cookieguru** [comment]
Related: #1659

**Comment by mpost** [comment]
The issue is indeed related to #1659 as @cookieguru pointed out. The select event on the `CollectionView` will be removed in the future as it causes multiple problems. It is recommended to attach a `tap` listener on the cell itself.

That being said, the error has been introduceed with Tabris.js 2.6 where the `linkTap` event has been added. This is a [know Android bug](https://issuetracker.google.com/issues/36908602) but we have worked around it by using [BetterLinkMovementMethod](https://github.com/saket/Better-Link-Movement-Method) which actually also improves upon the shortcomings of the link mechanism itself.
